core/src/main/java/org/mskcc/oncotree/utils/TumorTypesUtil.java

```diff
@@ -137,33 +136,60 @@ public static InputStream getTumorTypeInputStreamFromGitHub(Version version) {
         return tumorTypes;
     }
 
-    private static Map<String, TumorType> parseFromRaw(InputStream inputStream, Version version) {
-        TsvParserSettings settings = new TsvParserSettings();
+    private static Map<String, TumorType> loadFromRepository(Version version) {
+        List<OncoTreeNode> oncoTreeNodes = oncoTreeRepository.getOncoTree();
+        Map<String, TumorType> tree = new HashMap<>();
 
-        //the line separator sequence is defined here to ensure systems such as MacOS and Windows
-        //are able to process this file correctly (MacOS uses '\r'; and Windows uses '\r\n').
-        settings.getFormat().setLineSeparator("\n");
-
-        // creates a TSV parser
-        TsvParser parser = new TsvParser(settings);
-
-        Map<String, TumorType> tumorTypes = new HashMap<>();
+        TumorType root = new TumorType();
+        root.setCode("TISSUE");
```

Comment:
We manually create the tissue node?  I think that it will be returned from topbraid. Maybe it currently is not showing up because the sparql query is requiring a parent (broader) link. But I think that in topbraid there *is* actually a parent for the tissue node .. called "OncotreeConcept" or something like that.